system-addon/content-src/components/Card/_Card.scss

```diff
@@ -0,0 +1,108 @@
+.card {
+  $card-width: $grid-unit * 2 + $base-gutter;
+  $card-height: 266px;
+  $card-preview-image-height: 122px;
+  $card-description-height: 34px;
+  $card-description-height-max: 200px;
+
+  background: $white;
+  display: inline-block;
+  margin-inline-end: $base-gutter;
+  width: $card-width;
+  border-radius: $border-radius;
+  height: $card-height;
+  box-shadow: inset $inner-box-shadow;
+  position: relative;
+  @include context-menu-button;
+
+  > a {
+    display: block;
+    color: inherit;
+    height: 100%;
+    outline: none;
+    position: absolute;
+  }
+
+  &:hover, &:focus, &.active {
+    outline: none;
+    @include fade-in;
+    @include context-menu-button-hover;
+
+    .card-title {
+      color: $link-blue;
+    }
+  }
+
+  .card-preview-image {
+    position: relative;
+    background-size: cover;
+    background-position: center;
+    background-repeat: no-repeat;
+    height: $card-preview-image-height;
+
+    > img {
+      width: 100%;
+      height: 100%;
```

Comment:
This will cause image distortion. Why use an image rather than`background-image` for `.card-preview-image`?